packages/core/src/accessanalyzer/vue/iamPolicyChecks.vue

```diff
@@ -225,19 +242,24 @@ const client = WebviewClientFactory.create<IamPolicyChecksWebview>()
 export default defineComponent({
     mixins: [saveData],
     data: () => ({
-        documentType: 'JSON Policy Language',
+        documentType: 'CloudFormation',
         policyType: 'Identity',
         resourcePolicyType: 'Identity',
```

Comment:
I would recommend renaming this to `referencePolicyType` (as it is used for referencePolicy in CheckNoNewAccess)